tests/Backend/CommonPart1/BucketsTest.php

```diff
@@ -113,8 +115,26 @@ public function testBucketDetail()
 
     public function testBucketEvents()
     {
-        $events = $this->_client->listBucketEvents($this->getTestBucketId());
-        $this->assertNotEmpty($events);
+        // create unique bucket name to be sure that bucket events are for this bucket
+        $bucketName = sprintf('BucketsTest_testBucketEvents-%s-%s', time(), rand(1000, 9999));
```

Comment:
tie buckety sa tam budu hromadit pretoze pri dalsom behu sa vytvori novy, mozes tam pouzit nieco taketo nie ?
```
$description = 'testBycketEvents';
$this->initEmptyBucket($this->getTestBucketName($description), $stage, $description);
```
a bude to uz aj paralel test ready

Reply:
upraveno, dik